tests/integration/Microsoft.Bot.ApplicationInsights.Core.Tests/ServiceResolutionTests.cs

```diff
@@ -17,6 +17,10 @@ public class ServiceResolutionTests
     {
         public ServiceResolutionTests()
         {
+            // Arrange
+            // _server = new TestServer(new WebHostBuilder()
```

Comment:
Why is there commented out code in this test? 

I understand the common unit test pattern of Arrange, but this doesn't seem to be doing that.

Reply:
Ok Chris, we will review it and let you know. Thanks!

Reply:
Hi @cleemullins, regarding this commented code, we noticed that after rebasing with master we omitted the change were these lines were removed (more info [here](https://github.com/Microsoft/botbuilder-dotnet/pull/1402/files#diff-a64743665931f831c4585e1396ba4c82L19)). We just updated this, deleting the commented lines.